Services/GEV/WBD/tests/GevWBDSuccessWPStornoTest.php

```diff
@@ -19,38 +19,14 @@ public function setUp() {
 									),$this->row_id);
 	}
 
-	public function success_xml_error() {
```

Comment:
Ich hab versucht den Fehler zu erzwingen der in dem Test weiter unten erwartet wurde und hab es nicht hinbekommen. Daher denk ich macht der Test keinen Sinn.